api/HTMLSelectElement.json

```diff
@@ -843,6 +843,54 @@
           }
         }
       },
+      "showPicker": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLSelectElement/showPicker",
+          "spec_url": "https://html.spec.whatwg.org/multipage/input.html#dom-select-showpicker",
+          "support": {
+            "chrome": {
+              "version_added": "121"
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
```

Comment:
FYI only the bug now says this is enabled in "nightly" so have updated this to `preview` in https://github.com/mdn/browser-compat-data/pull/21280